tests/bluetooth/tester/src/btp_ccp.c

```diff
@@ -871,3 +872,304 @@ uint8_t tester_unregister_ccp(void)
 {
 	return BTP_STATUS_SUCCESS;
 }
+
+/* Telephone Bearer Service */
+static uint8_t tbs_supported_commands(const void *cmd, uint16_t cmd_len, void *rsp,
+				       uint16_t *rsp_len)
+{
+	struct btp_tbs_read_supported_commands_rp *rp = rsp;
+
+	/* octet 0 */
+	tester_set_bit(rp->data, BTP_TBS_READ_SUPPORTED_COMMANDS);
+	tester_set_bit(rp->data, BTP_TBS_REMOTE_INCOMING);
+	tester_set_bit(rp->data, BTP_TBS_HOLD);
+	tester_set_bit(rp->data, BTP_TBS_SET_BEARER_NAME);
+	tester_set_bit(rp->data, BTP_TBS_SET_TECHNOLOGY);
+	tester_set_bit(rp->data, BTP_TBS_SET_URI_SCHEME);
+	tester_set_bit(rp->data, BTP_TBS_SET_STATUS_FLAGS);
+
+	/* octet 1 */
+	tester_set_bit(rp->data, BTP_TBS_REMOTE_HOLD);
+	tester_set_bit(rp->data, BTP_TBS_ORIGINATE);
+	tester_set_bit(rp->data, BTP_TBS_SET_SIGNAL_STRENGTH);
+
+	*rsp_len = sizeof(*rp) + 1;
+
+	return BTP_STATUS_SUCCESS;
+}
+
+static uint8_t tbs_remote_incoming(const void *cmd, uint16_t cmd_len, void *rsp, uint16_t *rsp_len)
+{
+	const struct btp_tbs_remote_incoming_cmd *cp = cmd;
+	char friendly_name[CONFIG_BT_TBS_MAX_URI_LENGTH];
+	char caller_uri[CONFIG_BT_TBS_MAX_URI_LENGTH];
+	char rcv_uri[CONFIG_BT_TBS_MAX_URI_LENGTH];
+	char uri_string[cp->data_len];
+	int err;
+
+	LOG_DBG("");
+
+	if ((cp->recv_len >= sizeof(rcv_uri) || cp->caller_len >= sizeof(rcv_uri)) ||
```

Comment:
```suggestion
	char recv_uri[CONFIG_BT_TBS_MAX_URI_LENGTH];
	char uri_string[cp->data_len];
	int err;

	LOG_DBG("");

	if ((cp->recv_len >= sizeof(recv_uri) || cp->caller_len >= sizeof(recv_uri)) ||
```

Suggest to use the more common `recv` abbreviation instead of `rcv`

Reply:
Fixed